content/en/docs/Reference/operator.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Operators"
+linkTitle: "Operators"
+weight: 3
+date: 2021-03-15
+description: >
+    What are Operators
+---
+
+## What is an Operator?
+
+Operators make it easy to manage complex stateful applications on top of Kubernetes. By [Extending the Kubernetes API](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) with CustomResourceDefinitions it is possible to create what we call Operator Projects where you will be able to create and define your own API's to represent your solutions on the cluster and manage them by looking for the desired state. For a better understanding, see the [Operator Pattern][operator-pattern].
```

Comment:
Why not use the definitions from the introduction blog post? Those are much more refined and pretty much approved for enterprise customers.